helpers/obfuscation_helper.py

```diff
@@ -0,0 +1,13 @@
+import bcrypt
+import os
+
+from helpers.log_helper import create_log
+
+logger = create_log('obfuscation_helper')
+
+
+def obfuscate(input):
+    logger.debug('Obfuscating input \'{}\''.format(input))
+    hash = bcrypt.hashpw(str(input).encode(),
+                         os.environ['BCRYPT_SALT'].encode())
```

Comment:
Don't see this in the config file, is it set somewhere alongside the AWS keys, etc?

Reply:
Whoops forgot to add it originally -- should be in the config file now.